src/NAppUpdate.Framework/Utils/NauIpc.cs

```diff
@@ -17,10 +17,10 @@ namespace NAppUpdate.Framework.Utils
 	/// Starts the cold update process by extracting the updater app from the library's resources,
 	/// passing it all the data it needs and terminating the current application
 	/// </summary>
-	internal static class NauIpc
+	public static class NauIpc

```

Comment:
I had to elevate the access here so that the tests would build. Not sure how `internal` was possible before?